src/components/CreateAnalysis/FileLookup.tsx

```diff
@@ -1,102 +1,93 @@
 import React, { useContext, useState, useEffect } from "react";
-import { Button, TextInput, DatePicker, Split, SplitItem, isValidDate } from '@patternfly/react-core';
+import { Button, DatePicker, Split, SplitItem, isValidDate } from "@patternfly/react-core";
 import { AppContext } from "../../context/context";
 import { CreateAnalysisTypes, DicomImagesTypes } from "../../context/actions/types";
 import CreateAnalysisService, { StudyInstance } from "../../services/CreateAnalysisService";
 
 const FileLookup = () => {
-    const { state: {dcmImages}, dispatch } = useContext(AppContext);
+  const { state: { dcmImages }, dispatch } = useContext(AppContext);
 
-    const [seriesInstanceUID, setSeriesInstanceUID] = useState<string>('');
-    const [minCreationDate, setMinCreationDate] = useState<string>('');
-    const [maxCreationDate, setMaxCreationDate] = useState<string>('');
+  const [minCreationDate, setMinCreationDate] = useState<string>('');
+  const [maxCreationDate, setMaxCreationDate] = useState<string>('');
 
-    useEffect(() => {
-      const filteredDcmImages = dcmImages.allDcmImages.filter(image => {
-        const imageCreationDate = Date.parse(image.creation_date.substring(0, 10));
-        return (seriesInstanceUID === '' || image.SeriesInstanceUID.includes(seriesInstanceUID)) &&
-        (minCreationDate === '' || imageCreationDate >= Date.parse(minCreationDate)) &&
+  useEffect(() => {
+    const filteredDcmImages = dcmImages.allDcmImages.filter(image => {
```

Comment:
Type was missing in the original code. Could you please double check the type and add the missing type to this line? Thanks.

```suggestion
    const filteredDcmImages = dcmImages.allDcmImages.filter((image: DcmImage) => {
```